src/unwinder/sentry_unwinder_dbghelp.c

```diff
@@ -7,7 +7,11 @@ sentry__unwind_stack_dbghelp(
     void *addr, const sentry_ucontext_t *uctx, void **ptrs, size_t max_frames)
 {
     if (!uctx && !addr) {
+#if _WIN32_WINNT && _WIN32_WINNT < 0x0600
+        return 0;
```

Comment:
The rest of the unwinder still works though? Would be good to document these limitations in the README as well.

Reply:
Looks like `RtlCaptureStackBackTrace` is available on XP's version of `ntdll.dll`.
Replaced it with call via `GetProcAddress`.

Testing pending.